apps/ui/src/components/App/Topnav.vue

```diff
@@ -88,34 +92,22 @@ watch(
 </script>
 
 <template>
-  <nav
-    class="border-b fixed top-0 inset-x-0 z-50 lg:left-[72px] flex items-center justify-between h-[72px] bg-skin-bg space-x-4 pr-4"
-    :class="{
-      'translate-x-[72px] lg:translate-x-0': uiStore.sidebarOpen
-    }"
-  >
+  <UiTopnav v-bind="$attrs" class="gap-4 pr-4">
     <div
       class="flex items-center h-full truncate"
       :class="{
         'lg:border-r lg:pr-4 lg:w-[240px] shrink-0': hasAppNav,
         'border-r pr-4 w-[240px]': hasAppNav && uiStore.sidebarOpen
       }"
     >
-      <button
-        type="button"
-        class="text-skin-link cursor-pointer lg:hidden ml-4"
-        @click="uiStore.toggleSidebar"
-      >
-        <IH-menu-alt-2 />
-      </button>
-
+      <slot name="toggle-sidebar-button" />
       <Breadcrumb
         :class="[
           'ml-4',
           { 'hidden lg:flex': searchConfig && !uiStore.sidebarOpen }
         ]"
-      >
-      </Breadcrumb>
+        @click="emit('navigated')"
```

Comment:
`navigated` seem confusing at first, we are not navigating when we click so maybe we should use different name? like `breadcrumbClick` ?

Reply:
Breadcrumb navigates as well, but it's handled inside of the component.

Reply:
I am using `navigated` instead of just `click`, to make it obvious that this event happen only for when we click on a link, and not just a random click inside the component